promscale/quick-start.md

```diff
@@ -0,0 +1,166 @@
+# Quick Start with Promscale
+You can use Docker Compose to get started with Promscale. Docker Compose is a
+tool for running multi-container applications on Docker defined in the compose
+file format. A compose file is used to define how the one or more containers
+that make up your application are configured. 
+
+## Before you begin
+* Ensure that you have [installed Docker Compose][docker-compose].
+* Create a directory named `docker-compose`.
+
+## Install Promscale with Docker Compose
+To install Promscale with Docker Compose you need to create a compose file and define the services and then run the Docker Compose.
+
+### Creating a compose file and defining the services
```

Comment:
@VineethReddy02 why are you telling people to create a docker compose file? Why aren't we providing one directly so folks only need to do docker-compose up and they are done?